src/browser.coffee

```diff
@@ -18,6 +17,10 @@ CoffeeScript.run = (code, options = {}) ->
   options.shiftLine = on
   Function(compile code, options)()
 
+CoffeeScript.register = ->
```

Comment:
This is confusing because `register` method should not be available in browser environment, thus it should not be one of the abstract method. But doing so will break browser test. 

Because coffee's browser test is not actually ran in a browser. It runs in node environment and the first call in the `runTest` method is `CoffeeScript.register()`. I guess this might be because there's no karma, PhantomJs nor headless chromium back then.

That's why I set it to a NOP implementation to pass the test for now. I could configure a test script that actually runs in browser with one of the methods mentioned above. But I think that should be a separate PR/issue for discussion since it requires adding more devDependencies.